packages/html/ds/src/text-input/text-input.schema.ts

```diff
@@ -42,6 +39,9 @@ export const textInputSchema = zod.object({
       description: 'Sets the unique ID for the input field.',
     })
     .optional(),
+  label: labelSchema.optional(),
+  hintText: hintTextSchema.optional(),
+  errorText: errorTextSchema.optional(),
 });
```

Comment:
lines 42,43 and 44 will give an error because there is no description provided

Reply:
I think the `hintText` and `errorText` should be renamed to `hint` and `error` because they are objects that contain not just the text of the components but also `size` and `classNames`

Reply:
Good catch!